starlette/responses.py

```diff
@@ -122,6 +124,9 @@ def set_cookie(
                 "none",
             ], "samesite must be either 'strict', 'lax' or 'none'"
             cookie[key]["samesite"] = samesite
+        if sys.version_info >= (3, 14) and partitioned:
+            cookie[key]["partitioned"] = True  # pragma: no cover
```

Comment:
We want an error if `partitioned` is used on <3.14.

Reply:
Can you help me point what line is not coveredby the tests?